### DISCUSSION TITLE
Consider changing the colors to be more descriptive

### DISCUSSION THREAD

**Discussion by Nicholaiii** [AdguardTeam/AdGuardHome#3956] [Ideas]
Currently the "Top clients" list shows colors based on the amount of requests - the more requests, the greener.
Clients with few requests are then being displayed as worse, which really doesn't make much sense. 
\
![image](https://user-images.githubusercontent.com/7280931/146647314-1925b12e-102e-444f-a649-79892dc6a104.png)

**Comment by ainar-g** [comment]
As far as I know, it is already changed to be single-colour in the new design that should land in v0.108, if everything goes to plan.
